### ISSUE TITLE
Tokenize doesn't manage multilang apos

### ISSUE THREAD

**Issue by vdcrea** [davidemiceli/sentiment-multilang#4]
In french, apostrophe is very common: "the university" is written "l'université". Tokenize function output "luniversité", and it s not a relevant value. It's pretty complicated to have a tokenizer for french and english where "'" can be used in "it's" or "isn't".

**Comment by davidemiceli** [comment]
Yes, it is a knowed bug, I did not work to find a solution about this, it was complicated...
